packages/amplication-client/src/Components/Notifecation.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from "react";
+import {
+  NovuProvider,
+  PopoverNotificationCenter,
+  NotificationBell,
+  IMessage,
+} from "@novu/notification-center";
+import ThemeContext from "../Layout/ThemeContext";
+import { useQuery, gql } from "@apollo/client";
+import * as models from "../models";
+
+type TData = {
+  me: models.User;
+};
+
+const THEME_DARK = "dark";
+const THEME_LIGHT = "light";
+
+export function Notifications() {
```

Comment:
I'm not sure if you're the address, but I would expect an empty stage for this component while there is no data. At the moment it seems like you're return null

Reply:
also, I expect that last comment will be present as notification note at the side of my screen and then will be part of the rest of the notification pul. is this part of the UX ?

Reply:
Actually, I copy&paste the react component from Novu's documentation and added the `onNotificationClick`.
https://docs.novu.co/docs/notification-center/react-components

What do you mean by notification note at the side of the screen? Like a toast notification? I don't think that Novu supports such a thing. If we want it, we need to add it on our own.

Reply:
The notification center itself has an empty state for when no notifications exist. (Attaching an image)
![CleanShot 2022-06-12 at 11 46 39](https://user-images.githubusercontent.com/8872447/173225120-c7ffcf98-21b6-4879-8ed7-8bb131c7f146.png)

Regarding the toastr, there is no toastr functionality available yet, but we can add a hook for you. So when a new notification received, you could create a floating note for it with your design-system. Let me know if that is something needed :)